lib/resque.rb

```diff
@@ -165,6 +165,12 @@ def prune_interval
     @prune_interval || DEFAULT_PRUNE_INTERVAL
   end
 
+  attr_writer :enqueue_front
+  def enqueue_front
+    return @enqueue_front unless @enqueue_front.nil?
+    @enqueue_front = false
+  end
```

Comment:
not sure I follow, would this replace `attr_writer` or both the `attr_writer` and enqueue_front method? if it's the latter, where do we set the default `@enqueue_front` value?